README.md

```diff
@@ -66,11 +40,48 @@ Build a small application exposing the following endpoints through a REST API:
 
 We'll evaluate:
  
- * the complexity & scalability of your algorithm
- * the readability of your code (including readability of your tests)
- * the correctness of the outputs of the API
- * you're ability to share your design choices and clearly weight pros & cons of alternative solutions
- * the overall quality of your written technical communication 
+ * The complexity & scalability of your algorithm
+ * The readability of your code (including readability of your tests)
```

Comment:
If I'm correct this is the first time we mention tests in the assignments, we should at least indicate that we expect test in the enumeration above, no ?

Reply:
It's part of our version of the assignment since a while 🤔

Reply:
So, I've read it from bottom to top and when reaching that sentence it stroke me that we didn't asked for it.

I'm probably biased by my teaching experience where anything you don't ask for explicitly is never done (any ambiguity is interpreted toward the least effort ... ) but what apply to students may not apply to job candidates.

Reply:
From my experience on assignment, candidates tend to do the same (if it's not asked, it doesn't exist) a lot more than we'd expect (especially for things like commenting, tests, ...). It really depends if you want to make it an "implicit filter" by only taking people senior enough to put them even when not asked.

Reply:
👍 let's make it explicit, saying something along the lines of "Add the tests you deem necessary to assert the correct behavior of your solution."